app/controllers/bookmarks_controller.rb

```diff
@@ -9,7 +9,9 @@ def index
 
   def new
     @bookmark            = Bookmark.new
-    @bookmark.name       = params[:query].to_s.strip.split(/\s| = |!|~|>|</)[0]
+    query                = params[:query].to_s.squish
```

Comment:
On Thu, Apr 7, 2016 at 3:02 PM, Ivan Necas <notifications@github.com> wrote:

> name = "Some very bad name"


not sure i follow? what exactly is the case? regardless, pushed to another
issue is fine.